src/app/api/user/verifyEmail/route.ts

```diff
@@ -0,0 +1,47 @@
+import { NextRequest, NextResponse } from 'next/server';
+import { User } from '@/models/userModel';
+import { dbConnect } from '@/db/dbConnect';
+import bcrypt from 'bcryptjs'; // Changed to bcryptjs
+import { z } from 'zod';
+
+// Connect the database
+dbConnect();
+
+// Defining zod schema (no export here)
+const ZodValidation = z.object({
+  email: z.string().email(),
+});
+
+// Post the data
+export async function POST(request: NextRequest) {
```

Comment:
@freelancer79 We have to get the email for params object. Not by a request body

Reply:
Could you explain a bit more. I thought you asked me to add a route.

Reply:
We do not get the user email as a params object for the route file but request body. Check out nextjs docs how to can add dynamic routes

Reply:
Alright got it. I'll complete this today.